tests/rptest/tests/node_folder_deletion_test.py

```diff
@@ -85,7 +85,9 @@ def test_deleting_node_folder(self):
         self.redpanda.stop_node(to_stop)
         self.redpanda.clean_node(to_stop)
         # start node back up
-        self.redpanda.start_node(to_stop, auto_assign_node_id=True)
+        self.redpanda.start_node(to_stop,
+                                 auto_assign_node_id=True,
+                                 omit_seeds_on_idx_one=False)
```

Comment:
dicussion: any reason why not making `omit_seeds_on_idx_one=False` by default?

Ideally, that would involve reviewing all `start_node()` calls and changing it to the explicit `True` where necessary, but maybe that could be cut short simply by changing the default and running CI with that?

Reply:
At this point we probably should. I imagine there aren't many `start_node()` calls that depend on starting a one-node cluster -- some tests explicitly avoid picking node 1 for that reason.